x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/bulk_enable.ts

```diff
@@ -10,7 +10,7 @@ import { UserAtSpaceScenarios, SuperuserAtSpace1 } from '../../../scenarios';
 import { FtrProviderContext } from '../../../../common/ftr_provider_context';
 import { getUrlPrefix, getTestRuleData, ObjectRemover } from '../../../../common/lib';
 
-const defaultSuccessfulResponse = { total: 1, errors: [], taskIdsFailedToBeEnabled: [] };
+const defaultSuccessfulResponse = { total: 1, rules: [], errors: [], taskIdsFailedToBeEnabled: [] };
```

Comment:
let's test this new property

Reply:
I'm testing it here `x-pack/plugins/alerting/server/rules_client/tests/bulk_enable.test.ts`. Did not want to repeat tests.

Reply:
I've added rules list to one of my integration test: 95390aa9918c36f401e46a56a311d5bf962148d5.
I am testing rules property in unit tests for bulkEnable more extensively. I think it's enough. And here we are testing more access rights and validation. What do you think?

Reply:
Let's test the error case as well